play/src/front/Phaser/Game/MapEditor/Tools/AreaEditorTool.ts

```diff
@@ -436,6 +436,7 @@ export class AreaEditorTool extends MapEditorTool {
 
     public handleAreaDeletion(id: string, areaData: AreaData | undefined): void {
         this.scene.getGameMapFrontWrapper().listenAreaDeletion(areaData);
+        this.scene.getGameMapFrontWrapper().areasManager.removeArea(id);
```

Comment:
1. Is this normal that this method is not called in `TrashEditorTool.handleAreaDeletion`.
2. Shouldn't we move the call to `areasManager.removeArea` inside `listenAreaDeletion`?